README.md

```diff
@@ -97,14 +97,14 @@ var filter = {
    */
   minDate: 1554636310165, // timestamp (in milliseconds since UNIX epoch)
   maxDate: 1556277910456, // timestamp (in milliseconds since UNIX epoch)
+  bodyRegex: '(.*)How are you(.*)', // content regex to match
```

Comment:
Nice. Could you also update the _description_ for these filters in the comment section above(line 93 and 97)?

Reply:
Good one, I'll update the description to make it more complete.